index.d.ts

```diff
@@ -7,7 +7,7 @@
  * @param options all info for this parsing operation
  * @param variant the optional \uFE0F ("as image") variant, in case this info is anyhow meaningful. By default this is ignored.
  */
-export type ParseCallback = (icon: string, options: object, variant: string) => string | false;
+export type ParseCallback = (icon: string, options: Required<TwemojiOptions>, variant: string) => string | false;
```

Comment:
```suggestion
export type ParseCallback = (icon: string, options: TwemojiOptions, variant: string) => string | false;
```

`Required` won't work with strict null checks enabled because the default `attributes` value is a function returning `null`

Even the default callback doesn't use all of the options, so just `TwemojiOptions` seems sufficient here